boards/arm/thingy91x_nrf9161/dts/bindings/nordic,thingy91-nrf52840-reset.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: GPIO used to reset nRF52840 on Thingy:91
```

Comment:
The file name and content needs to replace thingy91 with thingy91x and nrf52 with nrf53.